blueprints/component-test/files/tests/__testType__/__path__/__test__.js

```diff
@@ -27,8 +27,8 @@ test('it renders', function(assert) {
   assert.equal(this.$().text().trim(), 'template block text');<% } else if(testType === 'unit') { %>assert.expect(1);
 
   // Creates the component instance
-  var component = this.subject();
+  /*var component =*/ this.subject();
```

Comment:
I've seen this style used in other place, e.g. in ember-cli-mirage. If we don't like this style. We could assert that `component` is truthy so the variable is used and JSHint doesn't complain. I do that in the [ember-cli-mocha](https://github.com/switchfly/ember-cli-mocha/blob/v0.9.2/blueprints/component-test/index.js#L33) flavor of the default test.